api/plugins/mongoose.js

```diff
@@ -0,0 +1,30 @@
+import fp from 'fastify-plugin';
+import mongoose from 'mongoose';
+
+import { mongodb } from '../config/config.js';
+
+/**
+ * @param {import("../app").Fastify} fastify
+ */
+export default fp(async (fastify) => {
+  try {
+    mongoose.connection.on('connected', () => {
+      fastify.log.info('MongoDB: Connected');
+    });
+    mongoose.connection.on('disconnected', () => {
+      fastify.log.info('MongoDB: Disconnected');
+    });
+
+    await mongoose.connect(`mongodb://${mongodb.host}:${mongodb.port}/db`, {
```

Comment:
```suggestion
    await mongoose.connect(`mongodb://${mongodb.host}:${mongodb.port}/db`, {
```

J'aurais mis le `mongodb` dans un .env